src/test/java/org/owasp/validator/html/test/AntiSammyMalformedPolicy.java

```diff
@@ -0,0 +1,36 @@
+package org.owasp.validator.html.test;
+
+import static org.junit.Assert.fail;
+
+import java.io.InputStream;
+
+import org.junit.Test;
+import org.owasp.validator.html.Policy;
+import org.owasp.validator.html.PolicyException;
+
+public class AntiSammyMalformedPolicy {
+
+    @Test
+    public void testDirectConfigMalformedXMLNoSchemaValidation() throws Exception {
+        Policy.setSchemaValidation(false);
+        InputStream stream = AntiSammyMalformedPolicy.class.getResourceAsStream("/invalidPolicyMalformedXml.xml");
+        try {
+            Policy.getInstance(stream);
+            fail("Malformed XML, PolicyException expected ");
+        } catch (PolicyException e) {
+            // Expected behaivour
+        }
```

Comment:
Please invoke Policy.setSchemaValidation(true); at the end of this test case, and all others that invoke Policy.setSchemaValidation(false); first, so validation is on by default when the test case completes.

Reply:
I added it to the @after  method.
It's needed to set the default value again.